api/models/userDao.js

```diff
@@ -12,6 +12,38 @@ const createUser = async (name, email, password, point) => {
   );
 };
 
+const getUserByEmail = async (email) => {
+  const result = await appDataSource.query(
+    `SELECT
+    id,
+    name,
+    email,
+    password,
+    point
+    FROM users
+    WHERE email=?`,
+    [email]
+  );
+  return result[0];
+};
```

Comment:
`result[0]` 로 0번째 인덱스를 활용하는 것 보다, 구조 분해 할당을 하는 것이 깔끔해 보입니다-!
```suggestion
const getUserByEmail = async (email) => {
  const [result] = await appDataSource.query(
    `SELECT
    id,
    name,
    email,
    password,
    point
    FROM users
    WHERE email=?`,
    [email]
  );
  
  return result;
};
```